packages/salesforcedx-sobjects-faux-generator/test/integration/sObjectDescribe.test.ts

```diff
@@ -15,6 +15,7 @@ import {
 import * as util from './integrationTestUtil';
 
 const PROJECT_NAME = `project_${new Date().getTime()}`;
+const keyLocation = path.join(process.cwd(), PROJECT_NAME, 'server.key');
```

Comment:
After using jwt, further org commands expect a file called server.key in the directory. We can obfuscate the filename if we change the oAuth options, but we delete the key in the after() anyways.